//Single try block with multiple catch block

class Test3{
	public static void main(String[] args) {
		try{
			
		}
		catch(ArithmeticException e){
			
		}
		catch(ArrayIndexOutOfBoundsException e){
			
		}
	}
}